src/commands/hub-group-api.ts

```diff
@@ -0,0 +1,69 @@
+import { CliUx, Command, Flags } from "@oclif/core"
+import HubGroup from '../api/hub'
+
+export default class HubGroupApi extends Command {
```

Comment:
we can remove this class, unit test should be enough

Reply:
removed